woocommerce/woocommerce
# Add To Cart Form: hide stepper when product is sold individually

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes a bug reported by @tomxygen (thanks) with product with sold individually option enabled. When this option is enabled, the stepper should not be rendered, given that only one instance of the product can be added in the cart.

| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/c79c3ceb-2a6a-45d6-b6ab-f56291f0b248) | ![image](https://github.com/user-attachments/assets/1685495e-08f4-4d5b-9cf9-20e5fe704734)| 





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Edit a simple product
2. Enable `Sold Individually` option:
![image](https://github.com/user-attachments/assets/f5513079-4109-4053-8ccf-8636f0551185)
3. Edit the Single Product Template.
4. Ensure that you have the Add To Cart Form block added.
5. Click on it.
6. Open the block settings.
7. Change the `Quantity Selector` option to "Stepper".
8. Save.
9. Visit the edited simple product.
10. Ensure that the stepper isn't visible.
11. Visit another product.
12. Ensure that the stepper is visible



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Add To Cart Form: hide stepper when product is sold individually.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
